### DISCUSSION TITLE
Multiple cursors within a line (or, in general, more multiple cursor flexibility)

### DISCUSSION THREAD

**Discussion by blex-max** [helix-editor/helix#10917] [Ideas] [answered]
I love the in-built multiple cursor functionality of helix, it's the feature that attracts me most. However, I wish it were extended further. Most notably, I'd like to be able to put cursors to multiple _different_ words on a line, so I can then perform operations like adding surrounds and so on. I'm wondering if others might also appreciate this or other additional flexibility in the multiple cursor system, and if it's implementation is likely or possible :)

**Comment by kirawi** [comment, marked as answer]
https://github.com/helix-editor/helix/issues/3007

alternnatively you could do `s` `\w+`
